client/package.json

```diff
@@ -20,7 +20,7 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^1.1.5",
+    "react-scripts": "^4.0.0",
```

Comment:
There was [another PR](https://github.com/hatchways/team-boolean-autocrats/pull/45/files) upgrading this to `^4.0.3`. Will probably resolve in a merge conflict but wanted to give a heads up beforehand!

Reply:
Should I leave it at 4.0.0? Or updated it to 4.0.3? Since that PR has not been merged?

Reply:
Maybe consider upgrading it 👍🏽